example/package.json

```diff
@@ -72,6 +72,7 @@
   },
   "devDependencies": {
     "@aws-sdk/client-ecs": "^3.447.0",
+    "@aws-sdk/client-lambda": "^3.529.1",
```

Comment:
I think these should be regular `dependencies` instead of `devDependencies`. In my PR I moved client-ecs up. It's a little weird because these are exclusively(?) used in integration tests in the `example` package. Thoughts?

Reply:
I'm not sure that it matters, functionally as this isn't a deployed package.    Happy to move them if we think that's right.